core/vm/actor/builtin/types/miner/v2/monies.cpp

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "vm/actor/builtin/types/miner/v2/monies.hpp"
+
+#include "vm/actor/actor_method.hpp"
+#include "vm/actor/builtin/v2/miner/miner_actor.hpp"
+#include "vm/actor/builtin/v2/multisig/multisig_actor.hpp"
```

Comment:
What do you need from multis_actor.hpp ?

Reply:
fixed